system-test/datastore.ts

```diff
@@ -17,20 +17,33 @@ import {readFileSync} from 'fs';
 import * as path from 'path';
 import {after, before, describe, it} from 'mocha';
 import * as yaml from 'js-yaml';
-import {Datastore, Index} from '../src';
+import {Datastore, DatastoreOptions, Index} from '../src';
 import {google} from '../protos/protos';
 import {Storage} from '@google-cloud/storage';
 import {AggregateField} from '../src/aggregate';
 import {PropertyFilter, and, or} from '../src/filter';
 import {entity} from '../src/entity';
 import KEY_SYMBOL = entity.KEY_SYMBOL;
 
+const async = require('async');
+
+const SECOND_DATABASE_ID = 'multidb-test';
+
+async.each(
+    [
+      {
+        namespace: `${Date.now()}`,
+      },
+      {
+        databaseId: SECOND_DATABASE_ID,
```

Comment:
where is the default db created?

Reply:
I'm not sure I understand the question. The default db is already set up in the integration test project. This object here with databaseId and namespace is one of the parameters for the parameterized testing. The other parameter which is above it just contains namespace so in that case, when these options are used to initialize datastore then the default database is assumed to be used.